Cargo.toml

```diff
@@ -75,6 +76,9 @@ tokio-stream = { workspace = true, features = ["net"] }
 message-hub = ["astarte-device-sdk/message-hub"]
 systemd = ["dep:systemd"]
 forwarder = ["dep:edgehog-forwarder"]
+udev = ["dep:udev"]
```

Comment:
What do you think to keep these features as default? Also to keep retro-compatibility with who already include our device-runtime in their image build system.
default = ["udev", "wifiscanner", "zbus"]